includes/managers/controls.php

```diff
@@ -1174,4 +1180,34 @@ public function add_custom_attributes_controls( Controls_Stack $controls_stack )
 
 		$controls_stack->end_controls_section();
 	}
+
+	/**
+	 * Check if the current stack should be cleaned by the responsive control duplication mode.
+	 *
+	 * @param $stack
+	 * @return bool
+	 */
+	private function should_clean_stack( $stack ): bool {
+		if ( ! isset( $stack['responsive_control_duplication_mode'] ) ) {
+			return false;
+		}
+
+		$current_stack_responsive_control_duplication_mode = $stack['responsive_control_duplication_mode'];
```

Comment:
if this variable lives only in this scope, why does it need to be that long?

```suggestion
		$stack_duplication_mode = $stack['responsive_control_duplication_mode'];
```

(if you're concerned about searching, the array access should cover that IMO)